openapi/Makefile

```diff
@@ -1,9 +1,12 @@
 # Generate Go types from OpenAPI spec.
-.PHONY: all
-all:
-	echo "No default target."
 
-.PHONY: gen
-gen:
-	oapi-codegen --config=oapi.config.yaml --o=manifest.gen.go manifest.yaml
-	oapi-codegen --config=oapi.config.yaml --o=problem.gen.go problem.yaml
+MANIFEST_YAML = https://raw.githubusercontent.com/amp-labs/openapi/main/manifest/manifest.yaml
+PROBLEM_YAML = https://raw.githubusercontent.com/amp-labs/openapi/main/problem/problem.yaml
+
```

Comment:
can you also add a `gen` command that runs both `gen/manifest` and `gen/problem`?